rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cat.allocation/10_basic.yml

```diff
@@ -194,7 +194,7 @@
   - match:
       $body: |
             /^
-              ( \d                  \s+    #usually 0, unless some system index has been recreated before this runs
+              ( \d+                 \s+    #usually 0, unless some system index has been recreated before this runs
                 0                   \s+
```

Comment:
Should we change this one while we're at it? It's the last 0 left in the test.

Reply:
I didn't, because that one's in a `bytes=gb` request, so by my reckoning there'd need to be (at least) more than 512mb of unexpected indices to flip that to a 1, and it seemed impossible to believe we'd end up in that situation.

OTOH, I am a fan of consistency, so I could go either way.

Reply:
Ohh I missed that this was a gb request. Seems fine the way it is then.